lib/Modules/NpmCommands.js

```diff
@@ -93,6 +93,21 @@ class NpmCommands {
       return [];
     }
   }
+
+  static async listModules({ path }) {
+    const npmInstalled = await NpmCommands.isNpmInstalled();
+    if (!npmInstalled) {
+      Log(colors.red('✖ Could not execute npm list, please install npm globally'));
+      return false; // Return false to be safe
+    }
+
+    try {
+      const { err, stdout } = await exec('npm list', { cwd: path });
+      return true
+    } catch (err) {
+      return false
```

Comment:
Maybe we want to log the stderr (or err depending on the contents) to give the developer more information on what exactly is missing?

Reply:
I thought of that, but npm list produces a lot of output and I didn't want to create a parser for it.